test/unit/mdc-drawer/mdc-drawer.test.js

```diff
@@ -104,35 +73,36 @@ test('#set open false calls foundation.close', () => {
 });
 
 test('keydown event calls foundation.handleKeydown method', () => {
-  const {component} = setupTest();
+  const {component, drawer} = setupTest();
   component.foundation_.handleKeydown = td.func();
-  component.initialSyncWithDOM();
-  domEvents.emit(document, 'keydown');
+  drawer.querySelector('.demo-drawer-list-item').focus();
```

Comment:
Is there a reason we actually need the `demo-drawer-list-item` class in the fixture at all? Can we just use `mdc-list-item`? (I realize it was there prior to this PR)